mlr_frontend/src/pages/ResultPage/ResultPage.js

```diff
@@ -15,13 +15,21 @@ const ResultPage = () => {
   }
 
   function handleExportBtn() {
+    generatePDF();
     navigate('/export');
   }
 
+  function generatePDF() {
+    const report = new JsPDF('portrait', 'pt', 'a4');
```

Comment:
I think we should use 'letter' (as in U.S. letter) instead of a4, as U.S. letter is more commonly used in the States (you cant even print a4 pdfs using Georgia Tech's printer)

Reply:
Thanks Tommy! Just fixed that